recipes/simbody/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.5.3" %}
+
+package:
+    name: simbody
+    version: {{ version }}
+
+source:
+    fn: Simbody-{{ version }}.tar.gz
+    url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
+    sha256: 8005fbdb16c6475f98e13b8f1423b0e9951c193681c2b0d19ae5b711d7e24ec1
+
+build:
+    number: 0
+    skip: true    # [win and py35]
+    features:
+        - vc14    # [win and py35]
+
+# NOTE: This installs the minimum dependencies for Simbody, i.e., no
+# visualizer. I'm also not completely sure if blas and lapack are necessary for
+# the unix builds because they are supposedly included with Simbody. Once
+# FreeGLUT, Xi, and Xmu are avaliable via Anaconda.org (free or conda-forge)
+# they can be added.
+requirements:
+    build:
+        - cmake
+        - blas    # [not win]
+        - lapack  # [not win]
+        - vc 14   # [win]
+    run:
+        - blas    # [not win]
+        - lapack  # [not win]
+        - vc 14   # [win]
+
```

Comment:
@moorepants you can add a simple "file exists" test here. See an example in:

https://github.com/conda-forge/hdfeos2-feedstock/blob/master/recipe/meta.yaml#L26-L27